### ISSUE TITLE
Failed to provision, upgrade refused

### ISSUE THREAD

**Issue by DylanVerstraete** [threefoldtech/zos#1028]
On node: `72CP8QPhMSpF7MbSvNR1TYZFbTnbRiuyvq5xwcoRNAib`

```
error failed to provision reservation 472731-8 error="trying to upgrade workloads to new version. new workload content is different from the old one. upgrade refused"
```

**Comment by DylanVerstraete** [comment]
I think this issue is fixed in `v0.4.5-rc3`, since this issue did not show up anymore after this release was added on testnet.

**Comment by muhamadazmy** [comment]
I don't get why this is an issue, this is a very old reservation here that is failing to migrate (not sure why) but this is not  really a problem, it's only this reservation that is affected.

http://explorer.testnet.grid.tf/api/v1/reservations/workloads/472731

Note: I can't find this reservation on the node cache, and I think it's safe to ignore.
